web-components/src/components/header/HeaderDropdownItems.tsx

```diff
@@ -32,6 +32,7 @@ const useStyles = makeStyles(theme => {
 export type HeaderDropdownItemProps = {
   title: string;
   href: string;
+  pathname: string;
```

Comment:
I had to do some prop drilling with this... there is probably a cleaner solution, but this works for now